.travis.yml

```diff
@@ -12,6 +12,7 @@ script:
   - yarn run lint
   - yarn run test:unit
   - yarn run test:e2e
+  - yarn run build
```

Comment:
you actually don't need run when using yarn. It just does it if it doesn't match an internal command.